packages/libwacom.rb

```diff
@@ -2,34 +2,45 @@
 
 class Libwacom < Package
   description 'libwacom is a wrapper library for evdev devices.'
-  homepage 'http://linuxwacom.sourceforge.net/wiki/index.php/Libwacom'
-  version '0.28-0'
+  homepage 'https://github.com/linuxwacom/libwacom'
+  @_ver = 1.8
+  version @_ver
   compatibility 'all'
-  source_url 'https://downloads.sourceforge.net/linuxwacom/libwacom-0.28.tar.bz2'
-  source_sha256 'e7d632301288b221cb5af69b4c5e57fd062bafd9a9acd6f9ce271570103267ef'
+  source_url "https://github.com/linuxwacom/libwacom/releases/download/libwacom-#{@_ver}/libwacom-#{@_ver}.tar.bz2"
+  source_sha256 '2e8075e60bbef74fe9c3539b0a0080efab28912b2552784d8b54dbbf1aaa63e5'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-0.28-0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-0.28-0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-0.28-0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-0.28-0-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-1.8-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-1.8-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-1.8-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libwacom-1.8-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '90853f518d643be9a53e4d8edd6f826156cc64189807a3565bffda8411fe9636',
-     armv7l: '90853f518d643be9a53e4d8edd6f826156cc64189807a3565bffda8411fe9636',
-       i686: 'cdd52b3f2c8a40be5907213be1861308ed53194a25b7b3afa9ca7e365025e741',
-     x86_64: '2ffb5dbd3620667a758b6e66f42a339474cdc9363bb942a8a696bc952b486b0b',
+  binary_sha256({
+    aarch64: '7029db156ea3a27a9024f12071cb1fb99a54f233b1df0b3c5eb6174d5e5a54fd',
+     armv7l: '7029db156ea3a27a9024f12071cb1fb99a54f233b1df0b3c5eb6174d5e5a54fd',
+       i686: '9c1f8a3548b8d374e7da99311b9ecd88781989f605ef7fa4f3b703f5bd8e6cf7',
+     x86_64: 'bd25043dcd7c6affa57e04ac2c9b6abebefa03ca7c6f83cb5c64827562ccf770'
   })
 
   depends_on 'libgudev'
   depends_on 'eudev'
+  depends_on 'libevdev'
```

Comment:
Do we need both libgudev and libevdev?

Reply:
libgudev: a library providing GObject bindings for libudev
libevdev: a wrapper library for evdev devices

So they are different...

Reply:
They are both needed, as they come up as configure failures otherwise.)